WorkforceDataApi/src/WorkforceDataApi.DevUtils/Csv/EstablishmentsCsvDownloader.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Net.Http;
+
+namespace WorkforceDataApi.DevUtils.Csv;
+
+public class EstablishmentsCsvDownloader : IEstablishmentsCsvDownloader
+{
+    private const string BaseDownloadUri = "https://ea-edubase-api-prod.azurewebsites.net/edubase/downloads/public/";
+    private readonly IHttpClientFactory _httpClientFactory;
+
+    public EstablishmentsCsvDownloader(IHttpClientFactory httpClientFactory)
+    {
+        _httpClientFactory = httpClientFactory;
+    }
+
+    public async Task<string> DownloadLatest()
+    {
+        var httpClient = _httpClientFactory.CreateClient();
+        httpClient.BaseAddress = new Uri(BaseDownloadUri);
+
+        var filename = GetLatestEstablishmentsCsvFilename();
+        using var response = await httpClient.GetAsync(filename);
+        response.EnsureSuccessStatusCode();
+
+        var stream = await response.Content.ReadAsStreamAsync();
+        using var fs = new FileStream(filename, FileMode.Create);
```

Comment:
Without a directory specified, where do these files go?

Reply:
to the folder we are running from .. this is only used by the command line utils .. but I could make it use a folder in ProgramData if you'd be happier with that ?

Reply:
Amended so that we are passing in a folder